src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -18,12 +18,7 @@ export const blankSchema = { type: 'object', properties: {} };
 
 export const applicantNameDobSchema = {
   uiSchema: {
-    ...titleUI(
-      ({ formData }) =>
-        `${
-          formData.certifierRole === 'applicant' ? 'Your' : 'Applicant'
-        } name and date of birth`,
-    ),
+    ...titleUI('Applicant name and date of birth'),
```

Comment:
This and other `'your' : 'applicant'` logic is now irrelevant because we will always use third person in page headings etc.